testUpdateHostsFile.py

```diff
@@ -249,7 +250,7 @@ def test_freshen_update(self, _):
 
     def tearDown(self):
         BaseStdout.tearDown(self)
-        BaseStdout.tearDown(self)
+        # BaseStdout.tearDown(self)
```

Comment:
Couldn't we do this like this to avoid erroring ?
```python
while 1:
    try:
        BaseStdout.tearDown(self)
    except:
        break
```
Of course, if there are any exceptions to deal with, deal with them appropriately, and don't forget to `break` the loop in any case.

Reply:
Even if it should work, I think this is not necessary.
As far as I understand, the tearDown method closes the current standard output file (it was changed before calling the setUp method) and re-sets it to the system standard output.
Calling tearDown twice causes the closure of the system standard output, raising an error in the following.